components-api/src/main/java/pl/touk/nussknacker/engine/api/TypingFunction.java

```diff
@@ -0,0 +1,16 @@
+package pl.touk.nussknacker.engine.api;
+
+import cats.data.NonEmptyList;
+import cats.data.Validated;
+import pl.touk.nussknacker.engine.api.typed.typing.TypingResult;
+import scala.Option;
+import scala.Tuple2;
+import scala.collection.immutable.List;
+
+public interface TypingFunction {
+    List<Tuple2<String, TypingResult>> expectedParameters();
```

Comment:
I wonder if we need this? Now we can just parse function arguments by reflection?